src/test/resources/features/99a_dividends_qa_smoke.feature

```diff
@@ -7,14 +7,15 @@ Feature: Smoke downstream Dividends Journeys to DES in QA - Income Tax Submissio
 
   Scenario: Returning flow - Individual User with prior UK Dividends & Other Dividends, Review & change - Dividends Smoke
     When the user logs into the service with the following details
-      |Redirect url       |/2023/start                                          |
-      |Credential Strength|strong                                               |
-      |Confidence Level   |200                                                  |
-      |Affinity Group     |Individual                                           |
-      |Nino               |CC555555D                                            |
-      |Enrolment Key 1    |HMRC-MTD-IT                                          |
-      |Identifier Name 1  |MTDITID                                              |
-      |Identifier Value 1 |9876543210                                           |
+      # It should work for 2023, but it seems there is a problem with the API
```

Comment:
Is this something that will be investigated outside of this ticket?

Reply:
I think this service is not in our control. Ben is aware of this and I suppose he will take it to the team responsible.